lib/modules/platform/github/branch.ts

```diff
@@ -13,7 +13,6 @@ async function matchingBranches(
 ): Promise<string[]> {
   const { body: branches } = await githubApi.getJson(
     `/repos/${repo}/git/matching-refs/heads/${branchName}`,
-    { memCache: false },
```

Comment:
Can you explain why the memCache=false parts are no longer needed?

Reply:
And yes, no memory cache is the default now

Reply:
How can we be confident that we haven't _missed_ anything?

Reply:
Generally speaking, we can't. What we can is analyze HTTP stats and watch for repeating GET requests (sometimes, though, it's intended).